tests/devices/default_qubit/test_default_qubit.py

```diff
@@ -23,6 +23,8 @@
 
 from pennylane.devices import DefaultQubit, ExecutionConfig
 
+np.random.seed(0)
```

Comment:
Will this be removed?

Reply:
My understanding was we should add random seeds to failing tests due to pytest having changed the order in which tests are run? https://xanaduhq.slack.com/archives/C0271MF2M50/p1706711364050839?thread_ts=1706566778.466919&cid=C0271MF2M50

Reply:
Ok, sounds good!